.changeset/dirty-clocks-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+Payment Splitter: add payees length getter
```

Comment:
```suggestion
`PaymentSplitter`: Add `getPayeesLength()` function returning the 
```

Reply:
Sorry, just realized I didn't finish this suggestion. I'll send a commit with adjustments